index.html

```diff
@@ -52,22 +54,28 @@ <h2 class="social-links-heading">LET'S CONNECT</h2>
             <div>
               <ul class="social-links-container">
                 <li class="twitter">
-                  <a href="#"><img src="images/twitter.png" /></a>
+                  <a href="#"
+                    ><img src="images/twitter.png" alt="Twiter icon"
+                  /></a>
                 </li>
                 <li class="linked-in">
                   <a href="#"
-                    ><img class="linkedin-i" src="images/I.png" /><img
-                      src="images/N.png"
+                    ><img
+                      class="linkedin-i"
+                      src="images/I.png"
+                      alt="LinkedIn icon" /><img src="images/N.png"
                   /></a>
                 </li>
                 <li class="message">
-                  <a href="#"><img src="images/M.png" /></a>
+                  <a href="#"><img src="images/M.png" alt="Medium icon" /></a>
                 </li>
                 <li class="cat-icon">
-                  <a href="#"><img src="images/cat.png" /></a>
+                  <a href="#"><img src="images/cat.png" alt="GitHub icon" /></a>
                 </li>
                 <li class="thumbs-up">
-                  <a href="#"><img src="images/thumps.png" /></a>
+                  <a href="#"
+                    ><img src="images/thumps.png" alt="Angel List icon"
+                  /></a>
```

Comment:
- [ ] Great design :wink: but you have some accessibility issues that I think you should look into. Some users depend on screen readers for them to successfully navigate your page and without working links in your `a` tags, they will find it difficult to connect or contact you in case they want to. Kindly add links to your social media icons to improve the accessibility of your page. Here's an accessibility test that suggests you do the same :point_down: 

![image](https://user-images.githubusercontent.com/99715640/214355486-1623f9be-80ac-48ce-af02-9f465b716980.png)